astropy/timeseries/binned.py

```diff
@@ -4,12 +4,11 @@
 
 import numpy as np
 
-from astropy.table import groups, Table, QTable
-from astropy.time import Time, TimeDelta
 from astropy import units as u
-from astropy.units import Quantity
-
+from astropy.table import QTable, Table, groups
```

Comment:
Doesn't "g" come before "Q"?

Reply:
Classes come first

Reply:
Ooops. I think I did the opposite of that in https://github.com/astropy/astropy/pull/13764 and https://github.com/astropy/astropy/pull/13765 but CI didn't complain, so maybe it's ok.